src/common/k8s-api/kube-api.ts

```diff
@@ -38,6 +38,17 @@ import type { RequestInit } from "node-fetch";
 import AbortController from "abort-controller";
 import { Agent, AgentOptions } from "https";
 import type { Patch } from "rfc6902";
+import electron from "electron";
+
+const onceSystemResume = () => 
+  new Promise(resolve => {
+    const handler = () => {
+      electron.powerMonitor.removeListener("resume", handler);
+      resolve(true);
+    };
+
+    electron.powerMonitor.on("resume", handler);
```

Comment:
Why not "once"?

Reply:
What do you mean?

Reply:
I mean you do `electron.powerMonitor.on("resume", handler)` but then in the handler you call `removeListener` in `handler`. I think it would be clear if this code was just:

```ts
function onceSystemResumes(): Promise<true> {
  return new Promise(resolve => electron.powerMonitor.once("resume", () => resolve(true)));
}
```

Reply:
Oh, ok, sounds good